open_humans/views.py

```diff
@@ -458,6 +458,11 @@ def get_context_data(self, **kwargs):
                     in ['share_username', 'send_messages', 'share_sources',
                         'all_sources']]))
 
+        # Decided whether or not to show the toggle public visible button
+        show_toggle_visible_button = ((not project_member.revoked) and
+                                      project_member.authorized and
+                                      project_member.member.public_data_participant.enrolled)
```

Comment:
wait, this shouldn't matter, should it? All members have the ability to hide membership?

It's just an added complication for members that can perform public data sharing (on how files get shared publicly) but all members have the ability to hide membership status...

Reply:
oh, so it should just test on authorized, then?

Reply:
checking "not revoked" is also fine, it should also be true & doesn't hurt to include it.

but it shouldn't be filtering out people that didn't activate the "public data sharing" feature...